pandas/tests/extension/test_common.py

```diff
@@ -1,14 +1,35 @@
 import numpy as np
 import pytest
 
+from pandas.compat import StringIO
+
 from pandas.core.dtypes import dtypes
 from pandas.core.dtypes.common import is_extension_array_dtype
 
 import pandas as pd
 from pandas.core.arrays import ExtensionArray
+from pandas.core.arrays.integer import (
+    Int8Dtype, Int16Dtype, Int32Dtype, Int64Dtype, UInt8Dtype, UInt16Dtype,
+    UInt32Dtype, UInt64Dtype, integer_array)
 import pandas.util.testing as tm
 
 
+def make_data():
+    return (list(range(1, 9)) + [np.nan] + list(range(10, 98))
```

Comment:
why are you adding fixtures here?

Reply:
this file shouldn't have been committed, I was testing things locally and forgot that I shouldn't have added tests here.